src/App.css

```diff
@@ -1,3 +1,68 @@
+* {
+  box-sizing: border-box;
+}
+
 .App {
   text-align: center;
 }
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: blueviolet;
+  padding-right: 6%;
+  position: fixed;
+  top: 1%;
```

Comment:
- [ ] (Optional) Nice styling, I would like to advise you to make the top `0` instead of 1%. That way, the `background-color` of the `.container` will not be showing above the navbar. 👍

Reply:
Thanks, will do that.